src/app/core/templates/activities/activity-create/activity-create.component.ts

```diff
@@ -203,13 +219,19 @@ export class ActivityCreateComponent implements OnInit {
                     this.empty = true;
                 }
 
-                if (dataObj.fullQuestion.includes("#,#")) { this.wrongCharacters = true; }
-                if (dataObj.fullQuestion.includes("#;#")) { this.wrongCharacters = true; }
+                if (dataObj.fullQuestion.includes("#,#")) {
+                    this.wrongCharacters = true;
+                }
+                if (dataObj.fullQuestion.includes("#;#")) {
+                    this.wrongCharacters = true;
```

Comment:
Is this variable ever used? I don't see an error being thrown.

Reply:
yeah it is further down